patches/blockprotocol+0.0.1.patch

```diff
@@ -87,23 +116,37 @@ index 205630a..ccd3499 100644
  };
  
  export type BlockProtocolDeleteLinksFunction = {
-@@ -219,6 +259,7 @@ export interface JSONArray extends Array<JSONValue> {}
+@@ -185,7 +233,10 @@ export type BlockProtocolAggregateEntityTypesFunction = {
+ };
+ 
+ type BlockProtocolUpdateEntityTypesAction = {
++  accountId?: string;
+   entityId: string;
++  entityTypeId?: string;
++  entityTypeVersionId?: string;
```

Comment:
We don't need these on `updateEntityTypes` because entity types are all of the same entity type (`EntityType`)

Reply:
Done.